packages/vue/src/components/atoms/SfBreadcrumbs/SfBreadcrumbs.vue

```diff
@@ -10,17 +10,23 @@
         <template v-if="last !== i">
           <!-- @slot Custom markup for previous pages (binds `breadcrumb` object) -->
           <slot name="link" v-bind="{ breadcrumb, go }">
-            <a class="sf-breadcrumbs__breadcrumb" @click="go(breadcrumb)">{{
-              breadcrumb.text
-            }}</a>
+            <a
+              v-focus
+              tabindex="0"
```

Comment:
In this case, we properly use tabindex because we don't have `href` attribute here, so focus won't work. But IMO we need here `href` attribute to provide valid html or change it to button because of the click event.  @pspaczek need your attention

Reply:
@martaradziszewska we should create SfLink element, something like on SfProductCart where you have <component is=> and you can set a, RouterLink or NuxtLink. On SPA we shouldn't have <a> and use link's for all elements with change route :) similar case is SFPagination